src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java

```diff
@@ -73,6 +81,42 @@ public boolean isOperationsUser(RestRequest request) {
         }
         return allowed;
     }
+    
+    @SuppressWarnings("rawtypes")
+    public void assertUser(RestRequest request) throws Exception {
+        final String user = getUser(request);
+        final String token = getBearerToken(request);
+        ConfigBuilder builder = new ConfigBuilder().withOauthToken(token);
+        try (DefaultOpenShiftClient osClient = new DefaultOpenShiftClient(builder.build())) {
+            LOGGER.debug("Verifying user {} matches the given token.", user);
+            Request okRequest = new Request.Builder()
+                .addHeader(AUTHORIZATION_HEADER, "Bearer " + token)
+                .url(osClient.getMasterUrl() + "oapi/v1/users/~") 
+                .build();
+            String username = null;
```

Comment:
Shouldn't `username` be declared outside the `try` block scope so that it will be in-scope for line 113?

Reply:
No.  On line 113 there is an explicit comparison that is outside the try block.